setup.py

```diff
@@ -1,11 +1,7 @@
-# -*- coding: utf-8 -*-
 from setuptools import setup
 
 
 setup(
-    extras_require={
```

Comment:
The PR refers to removal of Python2 code , this is still needed for Python3 versions before 3.5

I understand that they are all EOL, but it goes beyond the PR scope and preserving their support is not as expensive